doc/req.md

```diff
@@ -0,0 +1,143 @@
+<!---
+ Copyright (c) 2016 StrongLoop, IBM, and Express Contributors
+ License: MIT
+-->
+
+<h2>Request</h2>
+
+The `req` object represents the HTTP request and has properties for the
+request query string, parameters, body, HTTP headers, and so on.  In this documentation and by convention,
+the object is always referred to as `req` (and the HTTP response is `res`) but its actual name is determined
+by the parameters to the callback function in which you're working.
+
+For example:
+
+~~~js
+app.get('/user/:id', function(req, res) {
+  res.send('user ' + req.params.id);
+});
+~~~
+
+But you could just as well have:
+
+~~~js
+app.get('/user/:id', function(request, response) {
+  response.send('user ' + request.params.id);
+});
+~~~
+
+<h3 id='req.properties'>Properties</h3>
+
+<div class="doc-box doc-notice" markdown="1">
+In Express 4, `req.files` is no longer available on the `req` object by default. To access uploaded files on the `req.files` object, use multipart-handling middleware like [busboy](https://www.npmjs.com/package/busboy), [multer](https://www.npmjs.com/package/multer), [formidable](https://www.npmjs.com/package/formidable), [multiparty](https://www.npmjs.com/package/multiparty), [connect-multiparty](https://www.npmjs.com/package/connect-multiparty), or [pez](https://www.npmjs.com/package/pez).
+</div>
+
+<section markdown="1">
+  {% include api/en/4x/req-app.md %}
```

Comment:
I'm not sure how we could do that and preserve the functionality of the doc.  Maybe @hacksparrow has some idea.  

However, again, I think in general, the understanding needs to be that these files are intended to create the API docs on http://expressjs.com.  That is their purpose.  If there are some things that don't work or make sense here, then that is OK.  Anyone who intends to edit or contribute to the docs _must_ understand that, or the results will ultimately be problematic.